### ISSUE TITLE
Build for mac M1?

### ISSUE THREAD

**Issue by wez** [kakashidinho/metalangle#85]
Could you add an arm/m1 build for mac to the release downloads?

I was idly looking at building it for myself just now, but the [instructions](https://github.com/kakashidinho/metalangle/blob/master/doc/DevSetup.md#getting-the-source) require python2 and I didn't feel like running through getting that installed!

**Comment by Berstanio** [comment]
Don't they have already a arm64 slice?

**Comment by wez** [comment]
Not for mac:

```
19:49 wez@mba ~/Downloads/MetalANGLE.dylib.mac
; lipo -info libGLESv2.dylib
Non-fat file: libGLESv2.dylib is architecture: x86_64
19:50 took 11s wez@mba ~/Downloads/MetalANGLE.dylib.mac
; file libGLESv2.dylib
libGLESv2.dylib: Mach-O 64-bit dynamically linked shared library x86_64
```

**Comment by Berstanio** [comment]
Ahh, you are right for the dylib download. Anything speaking against using the framework? This has both slices

**Comment by kakashidinho** [comment]
Yes, MetalANGLE.framework has both libEGL and libGLESv2 APIs in one package and it has both x86 and arm slices. You could use it if you feel like it. The dylibs are built using google's gn script and atm it doesn't support arm architecture yet (upstream repo might already has a update for this).

**Comment by wez** [comment]
My application uses cross-platform GL bindings that want to dynamically load libGL, so I can't use the framework version.

**Comment by Berstanio** [comment]
The framework in the releases are dynamic librarys too afaik. So couldn't you just dynamically load the the framework? Or extract the dynamic lib from the framework and load that?

**Comment by wez** [comment]
Ah, I see, I can copy the framework in and rename it to libEGL.dylib and load it that way
